jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/tx/DisposeSessionTransactionSynchronization.java

```diff
@@ -42,7 +45,7 @@ public void afterCompletion(int status) {
 	    try {
 	        manager.disposeRuntimeEngine(runtime);
 	    } catch (Throwable e) {
-	        // catch exception as it's only clean up and should not affect runtime
+	        logger.warn("Could not dispose runtime engine", e);
```

Comment:
Please do -- in general, there are very few (if any) dispose or cleanup operations that should be warn. We only use warn for operations that directly prevent or impair normal operation. 

If you can push -f (force) a fix to the pull request, that would be great! Thanks.